pkg/orchestration/docker.go

```diff
@@ -253,12 +256,32 @@ func runNode(
 	case "prerelease":
 		branch = "stage"
 	default:
-		branch = config.Version
+		if semverRegex.MatchString(config.Version) {
+			branch = "main"
+		} else {
+			branch = config.Version // TODO: remove when adc repo is deprecated
+		}
 	}
 	if err := audiusCli(dockerClient, host, "pull-reset", branch); err != nil {
 		return logger.Error(err)
 	}
 
+	// Configure tag (will replace branch configuration)
+	var tag string
+	switch config.Version {
+	case "prerelease", "edge", "current":
```

Comment:
will the differences between these be documented anywhere?

Reply:
yeah, I'm working with Sam to ensure our documentation will be up to date for audius-d configs